tests/setupconfig/test_auth.py

```diff
@@ -0,0 +1,185 @@
+from django.core.management import CommandError, call_command
+
+import pytest
+import requests
+from django_setup_configuration.exceptions import ConfigurationRunFailed
+
+from mozilla_django_oidc_db.models import (
+    OpenIDConnectConfig,
+    UserInformationClaimsSources,
+)
+from mozilla_django_oidc_db.setupconfig.bootstrap import AdminOIDCConfigurationStep
+
+from ..conftest import KEYCLOAK_BASE_URL
+
+
+@pytest.mark.django_db
+def test_configure(setup_config_full):
+    call_command("setup_configuration")
+
+    config = OpenIDConnectConfig.get_solo()
+
+    assert config.enabled
+    assert config.oidc_rp_client_id == "client-id"
+    assert config.oidc_rp_client_secret == "secret"
+    assert config.oidc_rp_scopes_list == ["open_id", "email", "profile", "extra_scope"]
+    assert config.oidc_rp_sign_algo == "RS256"
+    assert config.oidc_rp_idp_sign_key == "key"
+    assert config.oidc_op_discovery_endpoint == ""
+    assert (
+        config.oidc_op_jwks_endpoint
+        == f"{KEYCLOAK_BASE_URL}protocol/openid-connect/certs"
+    )
+    assert (
+        config.oidc_op_authorization_endpoint
+        == f"{KEYCLOAK_BASE_URL}protocol/openid-connect/auth"
+    )
+    assert (
+        config.oidc_op_token_endpoint
+        == f"{KEYCLOAK_BASE_URL}protocol/openid-connect/token"
+    )
+    assert (
+        config.oidc_op_user_endpoint
+        == f"{KEYCLOAK_BASE_URL}protocol/openid-connect/userinfo"
+    )
+    assert config.username_claim == ["claim_name"]
+    assert config.groups_claim == ["groups_claim_name"]
+    assert config.claim_mapping == {"first_name": "given_name"}
```

Comment:
actually, this bug should really be caught by form validation :thinking:

Reply:
Use the JSON Field from jsonform and that is quite annoying to use, but it works